.github/ISSUE_TEMPLATE.md

```diff
@@ -30,6 +30,9 @@ What behavior did you expect to observe?
 **Execution command:**
 <!-- e.g. `./foo 42`. If an input file is required, include it as well. -->
 
+**Associated Future Test(s):**
+<!-- Are there any tests in Chapel's test system that demonstrate this issue?
+     e.g. test/path/to/foo.future #1234 -->
```

Comment:
> If it's a field that only makes sense to developers it's an early distraction / confusion for users.

This was my thinking for the placement, but I'm not attached to the decision. If we move it up, maybe it ought to include the prefix: "Optionally,"